applications/common/backend/src/main/resources/libs/kyanite/common/components/title/dialog/.content.json

```diff
@@ -48,11 +48,26 @@
 					"name": "element"
 				}
 			},
-			"anchorId": {
-				"sling:resourceType": "wcm/dialogs/components/textfield",
-				"name": "anchorId",
-				"label": "AnchorId text"
+			"overrideAnchorId": {
+				"sling:resourceType": "wcm/dialogs/components/toggle",
+				"name": "overrideAnchorId",
+				"label": "Override anchorId"
```

Comment:
Let's add the description: "The default anchor is created based on the title. However, authors can override it."

Reply:
You are right, we should inform the authors about its behaviour. I added it.